www/docs/en/6.x/config_ref/images.md

```diff
@@ -135,6 +136,22 @@ different screen resolutions.
         <icon src="res/Windows/Wide310x150Logo.scale-240.png" width="744" height="360" />
     </platform>
 ```
+
```

Comment:
Looks like these sizes are still [supported for Windows 8.1 apps](https://msdn.microsoft.com/en-us/library/windows/apps/xaml/hh781198.aspx#details_on_sizing__padding__scaling__and_transparency_for_windows_phone_store_apps)